olive/passes/onnx/append_pre_post_processing_ops.py

```diff
@@ -84,7 +84,7 @@ def _run_for_config(
     ) -> ONNXModel:
         from onnxruntime import __version__ as OrtVersion
 
-        output_model_path = ONNXModel.resolve_path(output_model_path)
+        output_model_path = ONNXModel.resolve_path(output_model_path, Path(model.model_path).name)
```

Comment:
Can you update `ONNXModel.resolve_path` to make sure it always has `.onnx` suffix in the `model_filename` part? 

All of these calls assume `Path(model.model_path).name` has `.onnx` suffix. It is true for all cases where olive creates the initial onnx model but it doesn't hurt to be safe.

Reply:
done.